dot/state/storage.go

```diff
@@ -70,11 +70,10 @@ func NewStorageState(db chaindb.Database, blockState *BlockState, t *trie.Trie)
 	tries[t.MustHash()] = t
 
 	return &StorageState{
-		blockState:    blockState,
-		tries:         tries,
-		baseDB:        db,
-		db:            chaindb.NewTable(db, storagePrefix),
-		subscriptions: make(map[byte]*StorageSubscription),
```

Comment:
does observerList need to be initialized? ie `[]Observer{}` I assume not since the tests work?

Reply:
add initialization